core/domain/wipeout_domain.py

```diff
@@ -37,45 +37,50 @@ def __init__(
             user_id,
             email,
             role,
-            normalized_username,
+            normalized_long_term_username,
             deletion_complete,
             pseudonymizable_entity_mappings):
         """Constructs a PendingDeletionRequest domain object.
 
         Args:
             user_id: str. The ID of the user who is being deleted.
             email: str. The email of the user who is being deleted.
-            normalized_username: str|None. The normalized username of the user
-                who is being deleted. Can be None.
+            normalized_long_term_username: str|None. The normalized username of
+                the user who is being deleted. Can be None when the user was on
+                the Oppia site only for a short time and thus the username
+                wasn't that known.
```

Comment:
wasn't that known --> hasn't been well-established yet.

Reply:
Done.